docs/user/introduction.asciidoc

```diff
@@ -1,12 +1,160 @@
 [[introduction]]
-== Introduction
+== {kib} &mdash; your window into the Elastic Stack
+++++
+<titleabbrev>What is Kibana?</titleabbrev>
+++++
 
-Kibana is an open source analytics and visualization platform designed to work with Elasticsearch. You use Kibana to
-search, view, and interact with data stored in Elasticsearch indices. You can easily perform advanced data analysis
-and visualize your data in a variety of charts, tables, and maps.
+**_Explore and visualize your data and manage all things Elastic Stack._**
 
-Kibana makes it easy to understand large volumes of data. Its simple, browser-based interface enables you to quickly
-create and share dynamic dashboards that display changes to Elasticsearch queries in real time.
+Whether you’re a user or admin, {kib} makes your data actionable by providing
+three key functions. Kibana is:
 
-Setting up Kibana is a snap. You can install Kibana and start exploring your Elasticsearch indices in minutes -- no
-code, no additional infrastructure required.
+* **An open-source analytics and visualization platform.**
+Use {kib} to explore your {es} data, and then build beautiful visualizations and dashboards.
+
+* **A UI for managing the Elastic Stack.**
+Manage your security settings, assign user roles, take snapshots, roll up your data,
+and more &mdash; all from the convenience of a {kib} UI.
+
+* **A centralized hub for Elastic's solutions.** From log analytics to
+document discovery to SIEM, {kib} is the portal for accessing these and other capabilities.
+
+[role="screenshot"]
+image::images/intro-kibana.png[]
+
+[float]
+[[get-data-into-kibana]]
+=== Getting data into {kib}
+
+{kib} is designed to use {es} as a data source. Think of Elasticsearch as the engine that stores
+and processes the data, with {kib} sitting on top.
+
+From the home page, {kib} provides these options for getting data in:
+
+* <<add-sample-data, Add a sample data set>> and take {kib} for a test drive without loading data yourself.
+* Set up a data flow between your app and the Elastic Stack using our built-in tutorials.
+(If a tutorial doesn’t exist for your data, go to the
+{beats-ref}/beats-reference.html[Beats overview] to learn about other data shippers
+in the {beats} family.)
+* Import log data using the file upload feature.
+* {ref}/getting-started-index.html[Index your data] in Elasticsearch.
```

Comment:
I know it's getting started, but some additional context might help here. If you're not using beats, then you're likely using our REST API or client libraries. "Index your data into Elasticsearch with REST API's or client libraries". That doesn't flow well, but gives a bit more context.